packages/config/src/projects/bobanetwork.ts

```diff
@@ -0,0 +1,204 @@
+      socialMedia: [
+        'https://blog.omgx.network',
+        'https://twitter.com/bobanetwork',
+        'https://t.me/omgnetwork',
+        'https://discord.com/invite/m7NysJjKhm'
+      ],
+    },
+    riskView: {
+      stateValidation: {
+        value: 'Proofs disabled',
+        description:
+          'Currently the system permits invalid state roots. More details in project overview.',
+        sentiment: 'bad',
+      },
+      dataAvailability: RISK_VIEW.DATA_ON_CHAIN,
+      upgradeability: RISK_VIEW.UPGRADABLE_YES,
+      operatorCensoring: RISK_VIEW.CENSORING_TRANSACT_L1,
+      operatorDown: RISK_VIEW.DOWN_ESCAPE_MP,
+    },
+    technology: {
+      category: {
+        name: 'Optimistic Rollup',
+      },
+      stateCorrectness: {
+        name: 'Fraud proofs are disabled',
+        description:
+          'Ultimately Boba Network will use fraud proofs to enforce state correctness. This feature is currently disabled and the system permits invalid state roots.',
+        risks: [
+          {
+            category: 'Funds can be stolen if',
+            text: 'an invalid state root is submitted to the system.',
+          },
+        ],
+        references: [
+          {
+            text: 'The incentive contract for verification proofs is disabled - Boba FAQ',
+            href: 'https://docs.boba.network/faq#the-incentive-contract-for-verification-proofs-is-disabled',
+          },
+        ],
+      },
+      dataAvailability: {
+        ...DATA_AVAILABILITY.ON_CHAIN_CANONICAL,
+        references: [
+          {
+            text: 'Data Availability Batches - Paradigm Research',
+            href: 'https://research.paradigm.xyz/optimism#data-availability-batches',
+          },
+        ],
+      },
+      operator: {
+        ...OPERATOR.CENTRALIZED_SEQUENCER,
+        references: [
+          {
+            text: 'Boba operates the only "Sequencer" node - Boba FAQ',
+            href: 'https://docs.boba.network/faq#boba-operates-the-only-sequencer-node',
+          },
+        ],
+      },
+      forceTransactions: {
+        ...FORCE_TRANSACTIONS.CANONICAL_ORDERING,
+        references: [
+          {
+            text: 'Chain Contracts - Boba documentation',
+            href: 'https://docs.boba.network/developer-docs/chain-contracts',
+          },
+        ],
+      },
+      exitMechanisms: [
+        {
+          name: 'Regular exit',
+          description:
+            'When a user initiates a withdrawal it is processed as a L2 to L1 message. Because Boba Network is an optimistic rollup this transaction has to be included in a block and finalized. This takes several days to happen after which the funds can be withdrawn on L1.',
+        },
+        {
+          name: 'Fast exit',
+          description:
+            'Users can initiate a fast exit which makes use of liquidity pools, and charges a small fee for the convenience. Users funds can then be withdrawn on L1 after only minutes.',
```

Comment:
Any references here? Also this paragraph could mention that the contract holding liquidity for liquidity pools only has a small amount of ETH

Reply:
Reference added. Note that the amount of ETH and other assets will grow as more people onboard and the TVL increases (as is already happening).